iktomi/cli/app.py

```diff
@@ -17,19 +17,19 @@
 
 try:
     MAXFD = os.sysconf("SC_OPEN_MAX")
-except:
+except: # pragma: no cover
     MAXFD = 256
 
 
-def close_fds(but=None):
+def close_fds(but=None):# pragma: no cover
```

Comment:
I believe it's time to drop this function and call `os.closarange()` in the only place it's used.